scripts/lib/install

```diff
@@ -127,7 +127,13 @@ if [ "$VIRTUALENV_NEEDED" = "yes" ]; then
     "$ZULIP_PATH"/scripts/lib/create-production-venv "$ZULIP_PATH"
 fi
 
-"$ZULIP_PATH"/scripts/lib/install-node
+if [ CIRCLECI ]; then
+    # in circleci not running this script as sudo cause
+    # an permission error when node and yarn are upgraded
+    sudo "$ZULIP_PATH"/scripts/lib/install-node
```

Comment:
It looks like you changed this to add a conditional for CI. That is the wrong direction. :-) Why does the non-`sudo` version give an error in CI and not in dev -- what is the difference between them that's causing that? Whatever the difference is, we should fix that; not make more differences.

Reply:
My understanding is the problem here is that `install-node` was run in the base image in Circle, and so what's failing here is running `install-node` where the version has changed.